pkg/util/util.go

```diff
@@ -71,3 +47,29 @@ func IsVersionGreaterThan(version1, version2 string) (bool, error) {
 	}
 	return semanticVersion2.LessThanOrEqual(semanticVersion1), nil
 }
+
+func PrintSandboxMessage() {
+	kubeconfig := strings.Join([]string{
+		"$KUBECONFIG",
+		f.FilePathJoin(f.UserHomeDir(), ".kube", "config"),
+		docker.Kubeconfig,
+	}, ":")
+
+	fmt.Printf("%v %v %v %v %v \n", emoji.ManTechnologist, progressSuccessMessage, emoji.Rocket, emoji.Rocket, emoji.PartyPopper)
+	fmt.Printf("Add KUBECONFIG and FLYTECTL_CONFIG to your environment variable \n")
+	fmt.Printf("export KUBECONFIG=%v \n", kubeconfig)
+	fmt.Printf("export FLYTECTL_CONFIG=%v \n", configutil.FlytectlConfig)
+}
+
+func GetRequest(url string) (*http.Response, error) {
+	client := &http.Client{}
+	req, _ := http.NewRequest("GET", url, nil)
+	response, err := client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	if response.StatusCode != 200 {
+		return nil, fmt.Errorf("someting goes wrong while sending request to %s. Got status code %v", url, response.StatusCode)
+	}
+	return response, nil
+}
```

Comment:
This can part of the requests_util which takes in url, method, body and return the response